exercises/F-logical-operators/exercise2.js

```diff
@@ -5,7 +5,23 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  return number < 0;
+
+}
+
+function isBetween5and10(number) {
+  return 5 < number < 10;
+}
+
+function isShortName(name) {
+  return name.length < 6;
```

Comment:
I think As Daniel has 6 Characters, in order to get the expected result, you need to have a bigger number here